spec/commands/admin/create_tokens_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module AnonymousCodes
+    module Admin
+      describe CreateTokens, type: :command do
+        let(:form) { double("FormObject", invalid?: false, num_tokens: 10) }
+        let(:code_group) { double("CodeGroup") }
+        let(:command) { described_class.new(form, code_group) }
+
+        describe "#call" do
+          context "when the form is valid" do
+            before do
+              allow(CreateTokensJob).to receive(:perform_later)
+              command.call
+            end
```

Comment:
```suggestion
        describe "#call" do
          before do
            allow(CreateTokensJob).to receive(:perform_later)
            command.call
          end
          context "when the form is valid" do
```
Let's move on the top so is valid for all examples

Reply:
Oh ok. Thanks.